calendar/calendar-tool/tool/src/webapp/vm/calendar/chef_calendar_view.vm

```diff
@@ -22,3 +22,8 @@
 <script src="/library/webjars/fullcalendar/5.8.0/main.min.js"></script>
 <script src="/library/webjars/fullcalendar/5.8.0/locales-all.min.js"></script>
 <script src="/sakai-calendar-tool/js/sakai-calendar.js"></script>
+
+<script>
+  const currentDate = '$currentDate';
+  calendar.gotoDate(currentDate);
```

Comment:
You may as well make this one line, unless you want to check that currentDate exists. What happens if you pass undefined or null to gotoDate?

Reply:
If you pass a non-parsable value it uses the browser current date!